modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -48,6 +48,67 @@
 #include <shlwapi.h>
 #endif
 
+static bool _get_blender_version(const String &p_path, int &r_major, int &r_minor, String *r_err = nullptr) {
+	String path = p_path;
+#ifdef WINDOWS_ENABLED
+	path = path.path_join("blender.exe");
+#else
+	path = path.path_join("blender");
+#endif
+
+#if defined(MACOS_ENABLED)
+	if (!FileAccess::exists(path)) {
+		path = path.path_join("Blender");
```

Comment:
This is a bit odd. I didn't want to delay the merge, but isn't the mac case-insensitive?

Reply:
APFS is case sensitive by default, but can be configured as case-sensitive: https://support.apple.com/guide/disk-utility/file-system-formats-dsku19ed921c/mac

Reply:
Does this work though?

This code tests `p_path + "/blender"`, and then if it fails it tests `p_path + "/blender/Blender"`.

Reply:
Yeah, that seems wrong. I can change it, but I don't have a mac to test on (again, this is moved code, not changed code)

Reply:
I've changed it, as it did seem most likely wrong, but don't have a mac to test on.